doc/users/colors.rst

```diff
@@ -7,26 +7,70 @@ Specifying Colors
 In almost all places in matplotlib where a color can be specified by the user
 it can be provided as:
 
-* ``(r, g, b)`` tuples
-* ``(r, g, b, a)`` tuples
-* hex string, ex ``#0F0F0F``, or ``#0F0F0F0F`` (with alpha channel)
-* float value between [0, 1] for gray level
-* One of ``{'b', 'g', 'r', 'c', 'm', 'y', 'k', 'w'}``
-* valid CSS4/X11 color names
-* valid name from the `xkcd color survey
-  <http://blog.xkcd.com/2010/05/03/color-survey-results/>`__ These
-  names are prefixed with ``'xkcd:'`` (e.g., ``'xkcd:sky blue'``) to
-  prevent name clashes with the CSS4/X11 names.
+* a ``(r, g, b)`` tuple
+* a ``(r, g, b, a)`` tuple
+* a hex string RGB or RGBA string (ex ``'#0F0F0F'`` or ``'#0F0F0F0F'``)
+* a float value in ``[0, 1]`` inclusive for gray level
+* one of ``{'b', 'g', 'r', 'c', 'm', 'y', 'k', 'w'}``
+* a X11/CSS4 color name
+* a name from the `xkcd color survey <http://xkcd.com/color/rgb/>`__
+  prefixed with ``'xkcd:'`` (e.g., ``'xkcd:sky blue'``)
+* one of ``{'C0', 'C1', 'C2', 'C3', 'C4', 'C5', 'C6', 'C7', 'C8', 'C9'}``
 
 All string specifications of color are case-insensitive.
 
-Internally, mpl is moving to storing all colors as RGBA float quadruples.
 
-There are 95 (out of 148 colors in the css color list) conflicts between the
-CSS4/X11 names and the xkcd names.  Given that the former are the standard
-color names of the web, matplotlib should follow them.  Thus, xkcd color names
-are prefixed with ``'xkcd:'``, for example ``'blue'`` maps to ``'#0000FF'``
-where as ``'xkcd:blue'`` maps to ``'#0343DF'``.
+``'CN'`` color selection
+------------------------
+
+Color can be specified by a string matching the regex ``C[0-9]``.
+This can be passed any place that a color is currently accepted and
+can be used as a 'single character color' in format-string to
+`matplotlib.Axes.plot`.
+
+This allows easy access to the color is the propriety cycle and to write code
+that will integrate with global changes to the style.  For example
+
+.. plot::
+   :include-source: True
+
+   import matplotlib as mpl
+   th = np.linspace(0, 2*np.pi, 128)
```

Comment:
`import numpy as np` and `import matplotlib.pyplot as plt` are implicit in the plot directive.  Probably should include them explicitly for copy-paste ability?